packages/server/src/api/routes/migrations.js

```diff
@@ -0,0 +1,14 @@
+const Router = require("@koa/router")
+const migrationsController = require("../controllers/migrations")
+const router = Router()
+const { internalApi } = require("@budibase/backend-core/auth")
+
+router
+  .post("/api/migrations/run", internalApi, migrationsController.migrate)
+  .get(
+    "/api/migrations/definitions",
+    internalApi,
```

Comment:
Should these be runnable in someway from the actual app? Right now looks like they'll be restricted to only being used on requests that have access to the API key.

Reply:
Right now that's the intention, so that authenticated users can't run migrations which is especially important for us in the cloud. 

I do like the idea of eventually moving stuff like this into the product though, similar to how updates can be invoked from the UI. A 'Server Management' page in the admin portal would be cool. Ideally you'd want to be able to see the live stream from the migration log there too, I'm not sure how we'd achieve it exactly in a multi clustered setup, might have to use s3 / minio as a log store which in itself would be a decent feature.

Reply:
If we wanted to do it without the API key we might need some other notion of a 'cluster admin' user that can perform tenant wide operations